unixfs/src/file/adder.rs

```diff
@@ -744,4 +744,36 @@ mod tests {
             "QmcHNWF1d56uCDSfJPA7t9fadZRV9we5HGSTGSmwuqmMP9"
         );
     }
+
+    #[test]
+    #[ignore]
+    fn full_link_block() {
+        let buf = vec![0u8; 1];
+
+        let mut adder = FileAdder::builder().with_chunker(Chunker::Size(1)).build();
+        let mut blocks_count = 0;
+
+        for _ in 0..174 {
```

Comment:
we really should have this magic number (which AFAIK is temporary) as a `const` somewhere at this point :smile:

Reply:
It does feel very natural for me having seen it for many months now, but if I use it with the builder to configure the layout then I dont have to make it a global :)